rllib/examples/attention_net.py

```diff
@@ -8,7 +8,8 @@
 from ray.rllib.examples.env.repeat_initial_obs_env import RepeatInitialObsEnv
 from ray.rllib.examples.env.stateless_cartpole import StatelessCartPole
 from ray.rllib.utils.framework import try_import_tf
-from ray.rllib.utils.test_utils import check_learning_achieved
+from ray.rllib.utils.test_utils import check_learning_achieved, \
```

Comment:
Fix all example scripts and Model classes in `examples/model` to work with env var RLLIB_FORCE_NUM_GPUS=[1|0|None].